### PR TITLE
Update Android ActionBar styles

### PR DESCRIPTION
**Pull request by thisandagain**
This PR brings the navigation styles as close spec as possible without creating a completely custom ActionBar.

### CODE REVIEW

**Comment by thisandagain** [comment]
@Pomax R?

File: app/src/main/java/mozilla/org/webmaker/activity/Tinker.java [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
     }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Custom styles
+        Resources res = getResources();
+        ColorDrawable plum = new ColorDrawable(getResources().getColor(R.color.plum));
+        int shadowPlum = res.getColor(R.color.shadow_plum);
```
Comment (Pomax) [inline app/src/main/java/mozilla/org/webmaker/activity/Tinker.java]: SHADOW PLUM

Comment (thisandagain) [inline app/src/main/java/mozilla/org/webmaker/activity/Tinker.java]: The most mysterious of the purples. :purple_heart: :ghost:

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
     <string name="title_section2">Make</string>
-
-    <string name="hello_world">Hello world!</string>
-
-    <string name="action_settings">Settings</string>
-    <string name="title_activity_map">Map</string>
-    <string name="title_activity_project">Project</string>
-    <string name="title_activity_editor">Editor</string>
-    <string name="title_activity_tinker">Tinker</string>
+    <string name="title_activity_map">Project</string>
```
Comment (Pomax) [inline app/src/main/res/values/strings.xml]: this is setting off my "clean refactor" senses. Either that activity name, or its content string, need to be brought in line,

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
+    <string name="title_activity_project">Page</string>
```
Comment (Pomax) [inline app/src/main/res/values/strings.xml]: same here...

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (thisandagain) [inline app/src/main/res/values/strings.xml]: Good call. I'll address that in #1649

**Comment by Pomax** [comment]
ya'R+ [[unflagged]](http://plzkthx.herokuapp.com)

**Pull request merged**